js/ui/form/ui.form.js

```diff
@@ -364,6 +367,13 @@ const Form = Widget.inherit({
         }
     },
 
+    _alignRootSimpleItems: function($container, colCount, labelsOptions) {
```

Comment:
Declaration: function($container, colCount, labelsOptions) 
Call: this._alignRootSimpleItems($container, colCount, excludeTabbed);

excludeTabbed -> labelsOptions

Can we keep the same name?

Reply:
Yep, fixed https://github.com/DevExpress/DevExtreme/pull/15045/commits/71209775b78eab2b61bbe037264852167903d785